test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -451,5 +477,92 @@ private static void DeleteDirectory(string path)
 
             }
         }
+
+        //temporary added methods for processing deps.json files for patching
+        private void TempPatching(string patchDir)
+        {
+            var prefix = patchDir;
+            string[] filenames = new string[3] { "dotnet.deps.json", "MSBuild.deps.json", "NuGet.CommandLine.XPlat.deps.json" };
+            string[] fullnames = new string[3];
+            for(int i = 0; i < filenames.Length; i++) 
+            {
+                fullnames[i] = prefix + Path.DirectorySeparatorChar + filenames[i];
+            }
+            TempPatchingDepsJsonForNewlyAddedDlls(fullnames);
```

Comment:
`fileNames` and `fullNames`.

You could also use LINQ to do this.

string[] fileNames = new [] { "dotnet.deps.json", "MSBuild.deps.json", "NuGet.CommandLine.XPlat.deps.json" };
string[] fullNames = fileNames.Select(fileName => Path.Combine(patchDirectoryPath, fileName)).ToArray();